gradlew

```diff
@@ -6,30 +6,12 @@
 ##
```

Comment:
@ruebot 
Not sure what was the process, what created a new PR.  https://github.com/Islandora-CLAW/Alpaca/pull/36.  Please delete/reject this PR.